frontend/src/components/task.jsx

```diff
@@ -1,29 +1,26 @@
 export default function Task(props) {
-    const { task, settasks } = props;
+    const { task, setTasks } = props;
 
-    const updatetask = async (taskId, taskStatus) => {
-        const res = await fetch(`backend/src/controllers/task.js${taskId}`, {
+    const updateTask = async (taskId, taskStatus) => {
+        await fetch(`http://localhost:3001/tasks/${taskId}`, {
             method: "PUT",
-            body: JSON.stringify({ status: taskStatus }),
+            body: JSON.stringify({ completed: taskStatus }),
             headers: {
                 "Content-Type": "application/json"
             },
         });
-
-        const json = await res.json();
-        if (json.acknowledged) {
```

Comment:
There's no `acknowledged` field in the response JSON, so we don't need to check for that.